Project_Mars/Project_Mars/StepDefinitions/LanguageStepDefinitions.cs

```diff
@@ -1,29 +1,38 @@
-using SpecFlow_Mars.Pages;
-using SpecFlow_Mars.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using Project_Mars.Pages;
+using Project_Mars.Utilities;
+using System;
 
 
-namespace SpecFlow_Mars.StepDefinitions
+namespace Project_Mars.StepDefinitions
 {
     [Binding]
-    public class LanguageStepDefinitions : CommonDriver
+    [Parallelizable]
+    public class LanguageStepDefinitions 
     {
         LoginPage LoginPageobj = new LoginPage();
```

Comment:
please use constructor here as you are initialising variables please see

 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors

Reply:
you have created a constructor but you have no made use of it properly